src/Microsoft.AspNet.SignalR.ServiceBus3/Infrastructure/ServiceBusTaskExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.AspNet.SignalR.ServiceBus.Infrastructure
+{
+    public static class ServiceBusTaskExtensions
```

Comment:
This an artifact of sharing code. This project has its own implementation of only ServiceBusConnection and SendAsync and all other code is from the other project. SendAsync is invoked from [ServiceBusConnectionContext](https://github.com/SignalR/SignalR/blob/1fba14fa3437e24c204dfaf8a18db3fce8acad3c/src/Microsoft.AspNet.SignalR.ServiceBus/ServiceBusConnectionContext.cs#L82) which is shared code. In the other project it is an [extension method](https://github.com/SignalR/SignalR/blob/1fba14fa3437e24c204dfaf8a18db3fce8acad3c/src/Microsoft.AspNet.SignalR.ServiceBus/Infrastructure/ServiceBusTaskExtensions.cs) so I just did the same here.